faq.md

```diff
@@ -281,6 +281,17 @@ is retained after a reload. This means you can quickly iterate
 on a screen deeply nested in your app, without starting
 from the home screen after every reload.
 
+### How is 'hot reload' different from 'full restart'?
```

Comment:
Do we also link to https://github.com/flutter/flutter/wiki/Hot-Reload or delete https://github.com/flutter/flutter/wiki/Hot-Reload ? (or, merge https://github.com/flutter/flutter/wiki/Hot-Reload into here?)